lib/RoadizCoreBundle/src/Routing/NodeUrlMatcher.php

```diff
@@ -29,7 +29,7 @@ public function __construct(
         PreviewResolverInterface $previewResolver,
         Stopwatch $stopwatch,
         LoggerInterface $logger,
-        private readonly NodeTypes $nodeTypesBag,
+        private readonly DecoratedNodeTypes $nodeTypesBag,
```

Comment:
Same here.

DecoratedNodeTypes must only be used when information is displayed to admins